features/choose_javascript_database_strategy.feature

```diff
@@ -37,13 +37,13 @@ Feature: Choose javascript database strategy
       """
       Feature:
         Background:
-          Given I have created 2 widgets
+          When I create 2 widgets
```

Comment:
I think `Given` works better here, to be honest, as `when` usually denotes an action that you do _given_ a state rather than set the state itself.

Reply:
if you look at what is happening, a `create` action is being invoked, which suggests it is an action. That was my 2 cents.

Reply:
Yep, I understand that. Just not sure about the lack of `Given`. I've always understood that `given` sets the initial state and `when` manipulates the state in a way that is verified by subsequent `then`steps.

I may have interpreted wrongly, though. :)